topics/proteomics/tutorials/metaquantome-data-creation/tutorial.md

```diff
@@ -0,0 +1,654 @@
+
+
+> ### {% icon hands_on %} Hands-on: Extracting biological processes for metaQuantome
+>
+> 1. {% tool [Filter](Filter1) %} with the following parameters:
+>    - {% icon param-file %} *"Filter"*: `Unipept_Function` (output of **Query Tabular** {% icon tool %})
+>    - *"With following condition"*: `c5=='biological process'`
+>    - *"Number of header lines to skip"*: `1`
+>
+> 2. Rename file as go_bp
+{: .hands_on}
+
+
+![Biological Processes](../../images/biologicalprocess.png){: width="60%"}
+
+## *Filter* - Cellular components
+
+This step is used to filter out the GO terms with cellular components and the corresponding number of peptides associated with these terms.
+
+> ### {% icon hands_on %} Hands-on: Extracting cellular component for metaQuantome
+>
+> 1. {% tool [Filter](Filter1) %} with the following parameters:
+>    - {% icon param-file %} *"Filter"*: `Unipept_Function` (output of **Query Tabular** {% icon tool %})
+>    - *"With following condition"*: `c5=='cellular component'`
+>    - *"Number of header lines to skip"*: `1`
+>
+> 2. Rename file as go_cc
+{: .hands_on}
+
+
+![Cellular-Component](../../images/cellularcomponent.png){: width="60%"}
+
+## *Filter* - Molecular Function
+
+This step is used to filter out the GO terms with molecular function and the corresponding number of peptides associated with these terms.
+
+> ### {% icon hands_on %} Hands-on: Extracting molecular function for metaQuantome
+>
+> 1. {% tool [Filter](Filter1) %} with the following parameters:
+>    - {% icon param-file %} *"Filter"*: `Unipept_Function` (output of **Query Tabular** {% icon tool %})
+>    - *"With following condition"*: `c5=='molecular function'`
+>    - *"Number of header lines to skip"*: `1`
+> 2. Rename file as go_mf
+{: .hands_on}
+
+
+![Molecular-Function](../../images/molecularfunction.png){: width="60%"}
+
+# **Conclusion**
+{:.no_toc}
+
+This completes the walkthrough of the metaQuantome data creation workflow .This tutorial is a guide to have datasets that are metaQuantome ready/compatible and can be used for metaproteomics research. We have incorporated only two conditions in this workflow but users can use as many as they want. Researchers can use this workflow with their data also, please make sure the tool parameters and the workflow will be needed to be modified accordingly.
+
+Please look at the following tutorials in this Metaproteomics series:
+
+[Metaproteomics]({% link topics/proteomics/tutorials/metaproteomics/tutorial.md %})
+This workflow is also available at [proteomics.usegalaxy.eu](https://proteomics.usegalaxy.eu/)
+
+This workflow was developed by the Galaxy-P team at the University of Minnesota. For more information about Galaxy-P or our ongoing work, please visit us at galaxyp.org
+
+> ### {% icon comment %} References
+>
+> - {% Batut2018 %}
```

Comment:
```suggestion
> - {% cite Batut2018 %}
```

Reply:
Need to add `cite` at the beginning of all the citation blocks here. This should allow the linter to pass.